analytics/terraform/spark-eks-ipv6/README.md

```diff
@@ -0,0 +1 @@
+# Spark K8s Operator running on Amazon EKS IPv6 cluster
```

Comment:
Add these hooks here


<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->